utils/GYBUnicodeDataUtils.py

```diff
@@ -252,7 +252,9 @@ def create_tables(self):
                 self.supp_data_offset_bits)
 
         # A mapping from BMP first-level index to BMP data block index.
-        self.BMP_lookup = [ i for i in range(0, 1 << self.BMP_first_level_index_bits) ]
+        # NOTE: That maps in Python 3 are not subscriptable. As a result the
+        #       BMP_lookup type has been changed from a map to a list.
+        self.BMP_lookup = list([ i for i in range(0, 1 << self.BMP_first_level_index_bits) ])
```

Comment:
The list comprehension should already return a list, so the outer `list()` is probably unnecessary